app/(auth)/logout/Logout.tsx

```diff
@@ -41,8 +41,10 @@ export default function Logout(props: Props) {
     const logout = async () => {
       await logoutHandler();
     };
-    logout();
-  }, [props.fakeSessionToken]);
+    logout().catch((error) => {
+      setOnError(error.message);
+    });
+  });
 
-  return null;
+  return <div>{onError}</div>;
```

Comment:
- this seems like the wrong thing inside the `{` and `}`
- maybe you should also write some more words here to indicate that this is an error message
- maybe try to fix it so that it does not possibly show a flash of "error" when logging out, if things load slowly

Reply:
https://github.com/upleveled/graphql-example-spring-2023-austria-vienna/pull/3/commits/392405da8767a9c701baff6f0da1b3387bc98781
https://github.com/upleveled/graphql-example-spring-2023-austria-vienna/pull/3/commits/caedcecfc3967f0984c1e193c026082ccb6a591b